packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Added screen-reader support for clinical-result icons. (Requires Jest test updates on consuming applications)
   * Added screen-reader support for alternative text for when no results are returned.
   * Added screen-reader support for clinical-result FlowsheetResultCell EnteredInError.
+  * Changed cells and header cells in clinical-result to use proper semantic html
```

Comment:
nit: Could we call out the FlowsheetResultCell, ResultNameHeaderCell and ResultTimeHeaderCell specifically?